api/src/main/resources/workbench-api.yaml

```diff
@@ -7278,7 +7278,7 @@ components:
         cdrVersionId:
           type: string
         creator:
-          type: string
+          $ref: '#/components/schemas/User'
```

Comment:
unfortunately this is not deployment-safe: when the API is deployed with this change, there's a period of time when the UI will be expecting a string here, before the UI is also updated.

Adding a field (maybe creatorUser?) and later deleting the original creator after a release would solve this issue.